src/utils/levelRouters/level1.js

```diff
@@ -14,6 +14,7 @@ const getComponentForLessonLevel1 = lessonNumber => {
     case 4:
     case 5:
     case 7:
+    case 8:
```

Comment:
@mark-yacoub  why the case numbers aren't ordered and there is no return here ?

Reply:
they are ordered, 7-8-9.

There is no return cause this a fall through. in a switch statement if you stack cases they all get the same implementation. this is way typically we put break or return to stop it falling through.